experimental/generation/runbot/TestBotHttpAdapter.cs

```diff
@@ -22,11 +22,9 @@ public class TestBotHttpAdapter : BotFrameworkHttpAdapter
             : base(configuration, credentialProvider)
         {
             this.UseStorage(storage);
-            this.UseState(userState, conversationState);
+            this.UseBotState(userState, conversationState);
             this.UseDebugger(configuration.GetValue("debugport", 4712), logger: logger);
 
-            HostContext.Current.Set<IConfiguration>(configuration);
```

Comment:
>HostContext.Current.Set<IConfiguration>(configuration); [](start = 12, length = 55)

If you don't set this luis settings won't work. 
    this.Use(new RegisterClassMiddleware<IConfiguration>(configuration));

Reply:
That is a bug I'm tracking down, so I will make that change.  
The issue was HostContext was not found.

---
In reply to: [423899800](https://github.com/microsoft/BotBuilder-Samples/pull/2378#discussion_r423899800) [](ancestors = 423899800)